pkg/lib/cockpit-components-plot.scss

```diff
@@ -1,64 +1,64 @@
 // Selected set of PF chart colors to optimize for full-spectrum and colorblindness
 // using unique part of PF name and a hex color fallback
 $plotColors: (
-    blue-300   #06c,
-    green-100  #bde2b9,
-    cyan-200   #73c5c5,
-    purple-100 #b2b0ea,
-    gold-300   #f4c145,
-    orange-300 #ec7a08,
-    red-200    #a30000,
-    cyan-300   #009596,
-    black-500  #4d5258
+  blue-300   #06c,
+  green-100  #bde2b9,
+  cyan-200   #73c5c5,
+  purple-100 #b2b0ea,
+  gold-300   #f4c145,
+  orange-300 #ec7a08,
+  red-200    #a30000,
+  cyan-300   #009596,
+  black-500  #4d5258
```

Comment:
Note: This is in a SCSS file and is correct for SCSS. Glancing over the commit made me wonder where the `:` are.. but this is right. :wink: It's a [SASS map](https://sass-lang.com/documentation/values/maps) with key value pairs.